src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -8,20 +8,40 @@ import loadLanguages from './loadLanguages'
 import loadCategories from './loadCategories'
 import loadEvents from './loadEvents'
 import fetchResourceCache from './fetchResourceCache'
+import moment from 'moment-timezone'
 
-export default function * loadCityContent (database: MemoryDatabase, newCity: string, newLanguage: string): Saga<void> {
-  if (database.hasContext(new MemoryDatabaseContext(newCity, newLanguage))) {
-    return // All data is already in the database
+const TWENTY_FOUR_HOURS = 24
+
+export default function * loadCityContent (
+  database: MemoryDatabase, newCity: string, newLanguage: string, forceRefresh: boolean): Saga<void> {
+  if (database.hasContext(newCity, newLanguage)) {
+    return
   }
 
   database.changeContext(new MemoryDatabaseContext(newCity, newLanguage))
 
+  yield call(database.readLastUpdate)
+
+  console.debug('Last city content update on ',
+    database.lastUpdate ? database.lastUpdate.toISOString() : database.lastUpdate)
```

Comment:
Couldn't this be changed to `database.lastUpdate && database.lastUpdate.toISOString()`?

Reply:
Could be, yes. It is just a debug statement though :)